window.c

```diff
@@ -195,7 +195,8 @@ add_to_window_list(rp_screen *s, Window w)
 	child_info = get_child_info(w, 1);
 	if (child_info) {
 		if (child_info->vscreen != new_window->vscreen)
-			new_window->vscreen = child_info->vscreen;
+			if (!defaults.win_new_use_vcur)
```

Comment:
Please combine these conditionals

Reply:
suggested name works for me, thanks for taking a look!